### PR TITLE
Style changes

### LINKED ISSUE #46: Norme everything

### PR DESCRIPTION
**Pull request by Eutienne**
close #46

### CODE REVIEW

File: src/client/client.cpp [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #include "client.hpp"
 
-Client::Client(const std::string& nick, TcpStream* stream) : _mNickName(nick), _mUserName(""), _mHostName(""), 
-_mServerName(""), _mRealName(""), _mStream(stream), _mAway(false), _mAwayMsg(""), _passwordUsedToConnect("")
+Client::Client(const std::string& nick, TcpStream* stream) : nickname_(nick), username_(""), hostname_(""), 
+servername_(""), realname_(""), stream_(stream), away_(false), awayMsg_(""), passwordUsedToConnect_("")
```
Comment (EvitaO) [inline src/client/client.cpp:4]: use lowerCamelCase also for nickname, hostname, servername, realname

File: src/client/client.hpp [hunk 2]
```diff
@@ -34,15 +34,15 @@ class Client
...
-        std::string _passwordUsedToConnect;
+        std::string nickname_;
+        std::string username_;
+        std::string hostname_;
+        std::string servername_;
+        std::string realname_;
+        TcpStream*  stream_;
+        bool        away_;
+        std::string awayMsg_;
+        std::string passwordUsedToConnect_;
```
Comment (EvitaO) [inline src/client/client.hpp:45]: also here and some other places

File: src/commands/cap_ls.cpp [hunk 3]
```diff
@@ -3,7 +3,8 @@
 #include <ctime>
 #include <algorithm>
 
-std::string gen_random(const int len) {
+std::string gen_random(const int len)
```
Comment (EvitaO) [inline src/commands/cap_ls.cpp:6]: use lowerCamelCase

File: src/commands/cap_ls.cpp [hunk 4]
```diff
@@ -12,7 +13,8 @@ std::string gen_random(const int len) {
     std::string tmp_s;
```
Comment (EvitaO) [inline src/commands/cap_ls.cpp:13]: weird variable name

File: src/connectable_client/connectable_client.hpp [hunk 5]
```diff
@@ -26,6 +19,12 @@ class ConnectableClient {
 
         Client& getClient();
 
+    private:
+        int             _port;
+        std::string     _host;
+        Client*         _client;
```
Comment (EvitaO) [inline src/connectable_client/connectable_client.hpp:25]: underscore on the wrong side?

File: src/parser/parser.cpp [hunk 6]
```diff
@@ -6,179 +6,180 @@ void    Parser::away(const std::string& str)
...
+    return command_;
 }
 
 std::string Parser::getRaw() const
 {
-    return this->_mRawText;
+    return this->rawText_;
 }
 
 std::string Parser::find_command(const std::string& s)
```
Comment (EvitaO) [inline src/parser/parser.cpp:185]: lowerCamelCase

**Comment by Eutienne** [comment]
#46

**Review by EvitaO — CHANGES_REQUESTED** [review]
see comments

**Review by EvitaO — APPROVED** [review]
good work!

**Pull request merged**